### DISCUSSION TITLE
Does pydantic contrib support Pydantic V2?

### DISCUSSION THREAD

**Discussion by plusiv** [tortoise/tortoise-orm#1419] [Q&A] [answered]
[Pydantic V2](https://github.com/pydantic/pydantic/releases/tag/v2.0.2) is out, and may be cool update the [contribution](https://tortoise.github.io/contrib/pydantic.html) to support and leverage the new versión 2 features.

**Comment by trankov** [comment]
The importance of this issue increases in the context that the new version of FastAPI uses Pydantic v2. This makes it impossible to use Tortoise ORM in the current version of FastAPI and beyond.

**Comment by long2ice** [comment, marked as answer]
Yes, that's important. I tried before, but which maybe big work.

**Reply by plusiv** [reply]
When will the release come out?
